tools/jit/templates/aten_dispatch.cpp

```diff
@@ -0,0 +1,69 @@
+#include "aten_dispatch.h"
+#include "torch/csrc/jit/interned_strings.h"
+#include "torch/csrc/utils/functional.h"
+
+#include <unordered_map>
+
+// ${generated_comment}
+
+namespace torch { namespace jit {
+
+using autograd::Variable;
+using autograd::variable_list;
+using at::Scalar;
+using at::Tensor;
+using at::IntList;
+using at::TensorList;
+using operator_constructor = std::function<TensorOp(jit::Node*)>;
+
+namespace {
+
+variable_list pack_list(Tensor v) { return { std::move(v) }; }
+variable_list pack_list(Scalar v) { return { v.toTensor() }; }
+variable_list pack_list(std::vector<Tensor> t) { return fmap<Variable>(t); }
+variable_list pack_list(std::tuple<Tensor, Tensor> v) {
+  return { std::move(std::get<0>(v)), std::move(std::get<1>(v)) };
+}
+variable_list pack_list(std::tuple<Tensor, Tensor, Tensor> v) {
+  return { std::get<0>(v), std::get<1>(v), std::get<2>(v) };
+}
+
+std::vector<Tensor> as_tensor_list(const variable_list& vars) {
+  return fmap(vars, [](Variable v) { return static_cast<Tensor>(v); });
```

Comment:
Lol we probably have, like, three implementations of this function scattered about in PyTorch, at this point.

Reply:
well it's simple enough :P